libfwupdplugin/fu-device.c

```diff
@@ -2607,6 +2607,10 @@ fu_device_fixup_vendor_name(FuDevice *self)
 	if (name != NULL && vendor != NULL) {
 		g_autofree gchar *name_up = g_utf8_strup(name, -1);
 		g_autofree gchar *vendor_up = g_utf8_strup(vendor, -1);
+		if (g_strcmp0(name_up, vendor_up) == 0) {
+			g_warning("name and vendor are the same: %s", name);
```

Comment:
this is kinda noisy to a user.  What are they gonna do about it?  How about only messaging for unsupported builds?

Reply:
Good idea! Fixed.